dali/test/python/operator_2/test_reduce.py

```diff
@@ -229,8 +228,8 @@ def test_reduce():
     for keep_dims in [False, True]:
         for reduction_name in reductions:
             for batch_gen in batch_gens:
-                for type_id in types:
-                    yield run_reduce, keep_dims, reduction_name, batch_gen, type_id
+                type_id = np.random.choice(types)
```

Comment:
all combinations of inputs is already tested in many more tests (the ones with overflow, type promotion, etc)